### PR TITLE
[Practice Exercises]: Add Better Error Handling Instructions & Tests for Error Raising Messages (# 5 of 8)

### PR DESCRIPTION
**Pull request by BethanyG**
Fifth set from [Issue #2563](https://github.com/exercism/python/issues/2563)

- [x] [nth-prime](https://github.com/exercism/python/tree/main/exercises/practice/nth-prime/) \| [test file](https://github.com/exercism/python/tree/main/exercises/practice/nth-prime/nth_prime_test.py) \| [template file](https://github.com/exercism/python/tree/main/exercises/practice/nth-prime/.meta/template.j2)
- [x] ~~[nucleotide-count](https://github.com/exercism/python/tree/main/exercises/practice/nucleotide-count/) \| [test file](https://github.com/exercism/python/tree/main/exercises/practice/nucleotide-count/nucleotide_count_test.py)~~ Exercise has been depreciated.
- [x] [ocr-numbers](https://github.com/exercism/python/tree/main/exercises/practice/ocr-numbers/) \| [test file](https://github.com/exercism/python/tree/main/exercises/practice/ocr-numbers/ocr_numbers_test.py) \| [template file](https://github.com/exercism/python/tree/main/exercises/practice/ocr-numbers/.meta/template.j2)
- [x] ~~[octal](https://github.com/exercism/python/tree/main/exercises/practice/octal/) \| [test file](https://github.com/exercism/python/tree/main/exercises/practice/octal/octal_test.py)~~ Exercise has been deprciated.
- [x] [palindrome-products](https://github.com/exercism/python/tree/main/exercises/practice/palindrome-products/) \| [test file](https://github.com/exercism/python/tree/main/exercises/practice/palindrome-products/palindrome_products_test.py) \| [template file](https://github.com/exercism/python/tree/main/exercises/practice/palindrome-products/.meta/template.j2)

### CODE REVIEW

File: exercises/practice/nth-prime/.meta/example.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 def prime(number):
     if number < 1:
-        raise ValueError('The parameter `number` has to be a positive number.')
+        raise ValueError('there is no zeroth prime')
```
Comment (J08K) [inline exercises/practice/nth-prime/.meta/example.py:7]: I mean I know this is an exemplar, but is `zeroth` really _that_ clear? :)

**Review by J08K — APPROVED** [review]
Just made one comment, but I don't think it is _that_ important for it to withhold you from merging. Great work! 💙

File: exercises/practice/nth-prime/.meta/example.py (hunk 1 above)
Comment (BethanyG) [reply exercises/practice/nth-prime/.meta/example.py:7]: You could always take it up in `problem-specifications`.  That wording came straight from the [`canonical.json`](https://github.com/exercism/problem-specifications/blob/00f45ee2f7666ab5d33f1ea18ae9b9a4db169821/exercises/nth-prime/canonical-data.json#L48).  😉

Comment (J08K) [reply exercises/practice/nth-prime/.meta/example.py:7]: Ah alrighty!

**Pull request merged**